samples/matter/lock/src/lock.matter

```diff
@@ -1778,7 +1778,7 @@ endpoint 0 {
     callback attribute generatedCommandList;
     callback attribute acceptedCommandList;
     callback attribute attributeList;
-    ram      attribute featureMap default = 1;
+    ram      attribute featureMap default = 0;
```

Comment:
I think that this should be marked as "callback" aka "external" as the default value doesn't really mean anything.